### PR TITLE
Initialized project.

### PR DESCRIPTION
**Pull request by patrycja93**
Created the project with basic dependencies. 
Added README file.

### CODE REVIEW

**Comment by afornalik** [comment]
Hey Patrycja,

I saw several maven configuration files in the PR. Maybe we should first add them to .gitignore on 'master' brach and then merge it with the 'main' branch, just to keep main branch clear. Please, look at this when you find time.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-configuration-processor</artifactId>
+			<optional>true</optional>
```
Comment (afornalik) [inline pom.xml:32]: nit: Do you know why we need this dependency? I read an article here -> https://www.baeldung.com/spring-boot-configuration-metadata and It looks that it is used for metadata generating. We may skip this for now.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	<name>animalshelter</name>
+	<description>Animal Shelter project to improve knowledge about Spring.</description>
+	<properties>
+		<java.version>11</java.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-actuator</artifactId>
+		</dependency>
```
Comment (afornalik) [inline pom.xml:23]: This dependency is not necessary for now.

**Review by afornalik — APPROVED** [review]
Some optional changes in the pom.xml file. LGTM

File: pom.xml (hunk 1 above)
Comment (patrycja93) [reply pom.xml]: Ok, deleted.

File: pom.xml (hunk 2 above)
Comment (patrycja93) [reply pom.xml:23]: I think, based on [this](https://www.javatpoint.com/spring-boot-actuator) article we will need this.

**Pull request merged**